cmd/oci-runtime-tool/generate.go

```diff
@@ -17,7 +17,7 @@ var generateFlags = []cli.Flag{
 	cli.StringFlag{Name: "apparmor", Usage: "specifies the the apparmor profile for the container"},
 	cli.StringFlag{Name: "arch", Value: runtime.GOARCH, Usage: "architecture the container is created for"},
 	cli.StringSliceFlag{Name: "args", Usage: "command to run in the container"},
-	cli.StringSliceFlag{Name: "bind", Usage: "bind mount directories src:dest:(rw,ro)"},
+	cli.StringSliceFlag{Name: "bind", Usage: "bind mount directories src:dest[:option1:option2:...]"},
```

Comment:
Well, I didn't find any rules about multiple options format in man7.
But from most of pages in man-pages, `[:options]` or `[:options...]` or `[:options]...` are all acceptable.